src/text_file_functions.cpp

```diff
@@ -641,7 +641,7 @@ QString AOApplication::get_char_shouts(QString p_char)
 {
   QString f_result = read_char_ini(p_char, "shouts", "Options");
   if (f_result == "")
-    return "default";
+    return current_theme; // The default option is the current theme.
```

Comment:
What happens if current_theme is not a thing?? Will it use the default theme in that case?

Reply:
Not sure what you mean by current_theme not being a thing.
It's an attribute of the AOApplication class, but I haven't check if that attribute is updated correctly every time it needs to.

Reply:
@Crystalwarrior
> What happens if current_theme is not a thing?? Will it use the default theme in that case?

Seems like it. See this search path: https://github.com/AttorneyOnline/AO2-Client/blob/ea17442682be48dcfe03cc44773b519be44ef78e/src/aomovie.cpp#L34-L48